packages/rule-sri/README.md

```diff
@@ -0,0 +1,145 @@
+# Require scripts and styles to use Subresource integrity (`sri`)
+
+`sri` warns about requesting scripts or styles without using Subresource
+integrity.
+
+## Why is this important?
+
+Nowadays it's very common to use third party resources from CDNs or different
+services (analytics, ads, etc.) and thus increasing the risk surface of your
+web application.
+
+While there are techniques to verify the agent is talking with the right server
+(TLS, HSTS, etc.), an attacker (or administrator) with access to the server can
+manipulate the content with impunity.
+
+> If you want to load a crypto miner on 1,000+ websites you don't attack 1,000+
+websites, you attack the 1 website that they all load content from.
+([Scott Helme][weak link])
+
+Subresource integrity [is a standard][sri spec] that mitigates this by ensuring
+that an exact representation of a resource, and only that representation, loads
+and executes.
```

Comment:
Maybe we should add a warning that this is only guaranteed over HTTPS?

Reply:
I put a comment in the latest comment in the next paragraph